dozer-cli/src/simple/executor.rs

```diff
@@ -94,6 +95,7 @@ impl<'a> Executor<'a> {
                 .map(|(endpoint, log)| (endpoint.clone(), Some(log.log.clone())))
                 .collect(),
             self.multi_pb.clone(),
+            Flags::default(),
```

Comment:
I think we'll need to pass flags from config here?

Reply:
That's right!